README.md

```diff
@@ -53,113 +53,132 @@ Follow each of the steps below to install automation-tools:
+    ```
 
 2. Copy the script:
 
-`cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh
-/etc/archivematica/automation-tools/`
+    ```
+    cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh /etc/archivematica/automation-tools/
+    ```
 
 3. Create a transfer source for the automation-tools in the storage service:
 
-You can use an existing transfer source, or set up a new one.
+    You can use an existing transfer source, or set up a new one. For Docker 
+    environments the standard transfer source is `$HOME/.am/ss-location-data/`.
 
 4. Your default script should be configured along the lines of this:
 
-`transfer-script.sh`:
-
-```bash
-#!/bin/bash
-# docker transfer script example
-# /etc/archivematica/automation-tools/transfer-script.sh
-cd /usr/lib/archivematica/automation-tools/
-python -m transfers.transfer \
- --am-url http://archivematica-dashboard:8000 \
- --ss-url http://archivematica-storage-service:8000 \
- --user test \
- --api-key test  \
- --ss-user test \
- --ss-api-key test \
- --transfer-source <transfer-source-in-storage-service> \
- --config-file transfers.conf
-```
+    `transfer-script.sh`:
+    ```bash
+    #!/bin/bash
+    # /etc/archivematica/automation-tools/transfer-script.sh
+    cd /usr/lib/archivematica/automation-tools/
+    # Use the Python in the virtual environment.
+    /usr/share/python/automation-tools/venv/bin/python -m transfers.transfer \
+     --am-url http://archivematica-dashboard:8000 \
+     --ss-url http://archivematica-storage-service:8000 \
+     --user test \
+     --api-key test  \
+     --ss-user test \
+     --ss-api-key test \
+     --transfer-source <transfer-source-in-storage-service> \
+     --config-file transfers.conf
+    ```
 
 5. Change permissions setting on the script:
 
-`chmod +x transfer-script.sh`
+    `chmod +x transfer-script.sh`
 
-This allows the script to be executible.
+    This allows the script to be executible.
 
-6. Run the script:
+6. Run `./transfer-script.sh` and you should have success!
 
-Run `./transfer-script.sh` and you should have success!
-
-Automated transfers
--------------------
-
-`transfers/transfer.py` is used to prepare transfers, move them into the
-pipelines processing location, and take actions when user input is required.
-Only one transfer is sent to the pipeline at a time, the scripts wait until the
-current transfer is resolved (failed, rejected or stored as an AIP) before
-automatically starting the next available transfer.
 
 Configuration
 -------------
 
-Suggested deployment is to use cron to run a shell script that runs the automate
-transfer tool. Example shell script (for example in
+Suggested deployment is to use cron to run a shell script that runs the 
+automation transfer tool. Example shell script (for example in
 `/etc/archivematica/automation-tools/transfer-script.sh`):
 
 ```bash
 #!/bin/bash
 cd /usr/lib/archivematica/automation-tools/
+# Use the Python in the virtual environment.
```

Comment:
Could possibly consider removing the definitive article before `Python` but is still clear otherwise.